src/conteiners/SearchInputContainer.js

```diff
@@ -0,0 +1,13 @@
+import { connect } from 'react-redux';
+import searchFilms from '../modules/actions/searchFilms';
+import SearchInput from '../components/SearchInput/SearchInput';
+
+
+
+const SearchInputContainer = connect(null, dispatch => ({
```

Comment:
the same
connection to redux should be based on redux hooks

Reply:
done